src/topojson/topojson.ml

```diff
@@ -248,56 +250,76 @@ module Make (J : Intf.Json) = struct
       geometry : geometry;
       properties : properties;
       foreign_members : (string * json) list;
+      id : id;
     }
 
-    let v ?(properties = `None) ?(foreign_members = []) geo =
-      { geometry = geo; properties; foreign_members }
+    let v ?(properties = `None) ?(foreign_members = []) ?(id = `None) geo =
+      { geometry = geo; properties; foreign_members; id }
 
     let geometry t = t.geometry
     let properties t = t.properties
     let foreign_members t = t.foreign_members
+    let id t = t.id
 
     let properties_of_json json =
       match J.find json [ "properties" ] with
       | Some j -> if J.is_null j then `Null else `Obj (decode_or_err J.to_obj j)
       | None -> `None
 
+    let id_of_json json =
```

Comment:
Same here -- the topojson specification doesn't make it clear what possible values the "id" should take, so we shouldn't narrow it down to just an object or null. `J.find json [ "id" ]` should suffice